stylesheets/commons/BigMatch.scss

```diff
@@ -1269,7 +1269,7 @@ span.slash {
 				opacity: 0.4;
 			}
 
-			.theme--dark & > img {
+			.wiki-leagueoflegends.theme--dark & > img {
```

Comment:
why only for lol?
would it possibly affect dota?
(atm no time to look up if dota uses this)